### PR TITLE
gradle file updates to publish to jcenter

### PR DESCRIPTION
**Pull request by kevinzetterstrom**
#### Ticket

Add to JCenter
#### Ticket Summary

Tweaking the gradle files to allow to publish to jcenter/bintray
#### Implementation Summary

Followed the docs...

### CODE REVIEW

**Comment by kvenn** [comment]
Looks good! Can you just throw one line into the confluence page under automation that we'll want to re-host those script files when we get around to it?
Will said he'll be using that doc to figure out how he's going to approach the build so it might be helpful.

File: build.gradle [hunk 1]
```diff
@@ -4,7 +4,8 @@ buildscript {
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:1.3.0'
```
Comment (Gadihoc) [inline build.gradle:6]: - [ ] [_@n_]()

File: vimeo-deeplink/build.gradle [hunk 2]
```diff
@@ -1,5 +1,30 @@
...
+    publishedGroupId = 'com.vimeo.android.deeplink'
+    libraryName = 'vimeo-deeplink'
+    artifact = 'vimeo-deeplink'
+
+    libraryDescription = 'A helper to deep link into the Vimeo Android app'
+
+    siteUrl = 'https://github.com/vimeo/vimeo-deeplink-android'
+    gitUrl = 'https://github.com/vimeo/vimeo-deeplink-android.git'
+
+    libraryVersion = '1.0.0'
```
Comment (Gadihoc) [inline vimeo-deeplink/build.gradle:17]: G@imageimageimage

File: vimeo-deeplink/build.gradle [hunk 3]
```diff
@@ -1,5 +1,30 @@
 apply plugin: 'com.android.library'
 
+
+ext {
```
Comment (Gadihoc) [inline vimeo-deeplink/build.gradle:4]: - [ ] - imageimageimageimage_[[]()]()_@**_trunglatrung_**

**Review by Gadihoc — COMMENTED** [review]
G

**Pull request merged**